frontend/src/i18n/locales/en.yaml

```diff
@@ -302,13 +302,24 @@ manage:
       user: User
       yourself: You
       subset-warning: 'This selection is already included in the following group(s): {{groups}}.'
+      inherited: Inherited permissions
+      inherited-tooltip: The following entries can only be changed on the page they originate from.
```

Comment:
I think I would write something slightly different. When I first read this tooltip just now, my initial reaction was "so why don't _you_ tell me from what page they originate, punk?!"

To be clear: I don't suggest we add that feature! :D I would reformulate this to make users less likely to want that feature :P What about:

> The following permissions were granted in a parent page and are inherited to all sub pages.

With one issue: "parent page". Technically it's "ancestor" instead of "parent", but even "parent" might not be sth users are familiar with? "Super page"? "a page higher up the hierarchy"?